src/Uno.UWP/ApplicationModel/Calls/PhoneCallHistoryManager.Android.cs

```diff
@@ -0,0 +1,145 @@
+﻿
+#if __ANDROID__
+
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Foundation;
+
+namespace Windows.ApplicationModel.Calls
+{
+	public partial class PhoneCallHistoryManager
+	{
+
+		private static async Task<PhoneCallHistoryStore> RequestStoreAsyncTask(PhoneCallHistoryStoreAccessType accessType)
+		{
+			// UWP: AppEntriesReadWrite, AllEntriesLimitedReadWrite, AllEntriesReadWrite
+			// Android: Manifest has READ_CALL_LOG and WRITE_CALL_LOG, no difference between app/limited/full
+			// using: AllEntriesReadWrite as ReadWrite, and AllEntriesLimitedReadWrite as ReadOnly
+
+			if(accessType == PhoneCallHistoryStoreAccessType.AppEntriesReadWrite)
+			{   // should not happen, as this option is defined as NotImplemented in enum
+				throw new NotImplementedException();
+			}
+
+
+			var _histStore = new PhoneCallHistoryStore();
+
+			// below API 16 (JellyBean), permission are granted
+			if (Android.OS.Build.VERSION.SdkInt < Android.OS.BuildVersionCodes.JellyBean)
+			{
+				return _histStore;
+			}
+
+			// since API 29, we should do something more:
+			// https://developer.android.com/reference/android/content/pm/PackageInstaller.SessionParams.html#setWhitelistedRestrictedPermissions(java.util.Set%3Cjava.lang.String%3E)
+
+			// do we have declared this permission in Manifest?
+			// it could be also Coarse, without GPS
+			Android.Content.Context context = Android.App.Application.Context;

```

Comment:
use `var` wherever possible.

```suggestion
			var context = Android.App.Application.Context;

```